bad_connector_woocommerce/components/backend_adapter.py

```diff
@@ -186,6 +186,9 @@ class GenericAdapter(AbstractComponent):
     _apply_on = "woo.backend"
     _last_update_date = "date_modified"
     _woo_model = None
+    _woo_default_currency = None
+    _woo_default_weight = None
+    _woo_default_dimension = None
```

Comment:
better to pass this by inheriting search method in kwargs and have condition based on it